indico/legacy/webinterface/tpls/RoomBookingNewBookingPeriodWidget.tpl

```diff
@@ -208,6 +217,47 @@ ${ form.repeat_interval(type='hidden') }
             }
         }
 
+        function checkBookingLimit() {
+            var oneDay = 24*60*60*1000;
```

Comment:
No strong opinion here except for adding space around the `*`, but I think everyone knows `86400` is a day so I'd have gone for `86400 * 1000`